Create workplace-program.md

jasonblais opened the pull request.

jasonblais committed:
Create workplace-program.md

jasonblais committed:
Update SUMMARY.md

justinegeffen commented on operations/workplace/people/working-at-mattermost/workplace-program.md:
```diff
@@ -0,0 +1,74 @@
+At Mattermost, we like to share celebrations of life milestones and sympathy with our team members!
+
+If you know someone who is either
+Celebrating a special milestone such as wedding engagements, weddings and childbirth; or
```
```suggestion
- Celebrating a special milestone such as wedding engagements, weddings, and childbirth; or
```

justinegeffen commented on operations/workplace/people/working-at-mattermost/workplace-program.md:
```diff
@@ -0,0 +1,74 @@
+
+If you know someone who is either
+Celebrating a special milestone such as wedding engagements, weddings and childbirth; or
+Undergoing a major life event such as the passing of a family member/significant other, or is injured, ill or hospitalized
```
```suggestion
- Undergoing a major life event such as the passing of a family member/significant other, or is injured, ill or hospitalized
```

justinegeffen commented on operations/workplace/people/working-at-mattermost/workplace-program.md:
```diff
@@ -0,0 +1,74 @@
+Notes:
+
+  - Multiple requests for the same event are combined into a single gift.
+  - All gifts will be sent to the address on file on their Bamboo profiles
```
```suggestion
  - All gifts will be sent to the address on file on their Bamboo profile.
```

justinegeffen commented on operations/workplace/people/working-at-mattermost/workplace-program.md:
```diff
@@ -0,0 +1,74 @@
+
+  - Multiple requests for the same event are combined into a single gift.
+  - All gifts will be sent to the address on file on their Bamboo profiles
+  - Before submitting a request, please confirm with the team member that their shipping address and phone number is currently up to date on their Bamboo profiles, and ask them to update it if it is not current.
```
```suggestion
  - Before submitting a request, please confirm with the team member that their shipping address and phone number is currently up to date on their Bamboo profile, and ask them to update it if it is not current.
```

justinegeffen reviewed (dismissed):
This great, thanks @jasonblais. :)

jasonblais committed:
Update operations/workplace/people/working-at-mattermost/workplace-program.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

jasonblais committed:
Update operations/workplace/people/working-at-mattermost/workplace-program.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

jasonblais committed:
Update operations/workplace/people/working-at-mattermost/workplace-program.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

jasonblais committed:
Update operations/workplace/people/working-at-mattermost/workplace-program.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

jasonblais committed:
Add how to join video game activities

jasonblais committed:
Update workplace-program.md

hannaparks commented:
Hey @jasonblais, looks great!

Would we be able to get the yoga budget approval before this page goes live?

jasonblais commented:
> Would we be able to get the yoga budget approval before this page goes live?

Yes, or alternatively we can publish without the yoga session if it'll need some extra time.

hannaparks commented:
> > Would we be able to get the yoga budget approval before this page goes live?
> 
> Yes, or alternatively we can publish without the yoga session if it'll need some extra time.

No worries, let's keep it in there!

natalie-hub committed:
Update workplace-programs.md

I changed "build" to "foster" because we can't take ownership of building friendships but we can try and create ways that promote the build or development of friendships. 1/5 on this.

natalie-hub commented:
I made one word change suggestion, but otherwise this looks great! Thank you @hannaparks and @jasonblais for putting together and documenting!

natalie-hub reviewed (approved):
I'm so sorry, I should have added my update to the word here. The changes look good!

jasonblais committed:
Update workplace-program.md

hannaparks commented on operations/workplace/people/working-at-mattermost/workplace-program.md:
```diff
@@ -0,0 +1,74 @@
+
+### Video Game Hour
+
+Into video games? Then join your fellow team to battle for the best at 11:30am Palo Alto time each Friday.
```
Hey @jasonblais, we need to change the time to 10:30am Palo Alto time.
